app/controllers/HomeController.java

```diff
@@ -1,21 +1,58 @@
 package controllers;
 
+import LunchManCore.Apprentice;
+import LunchManCore.FridayLunch;
+import LunchManCore.Rota;
 import play.mvc.*;
 
+import services.CSVHelper;
 import views.html.*;
 
-import java.util.Arrays;
+import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+
+import static services.CSVHelper.createScheduleFromCSV;
 
 /**
  * This controller contains an action to handle HTTP requests
  * to the application's home page.
  */
 public class HomeController extends Controller {
 
+    private String apprenticeCSV = "/Users/molliestephenson/Java/LunchMan/csvs/apprentices.csv";
```

Comment:
We experienced an issue yesterday in that when trying to run tests within IntelliJ the base of the relative path was different to when running it from gradle. So the app would work in development, but not for testing within the IDE. We weren't able to find a quick solution to this at the time.